pkg/vmimage/data/usr/local/bin/get-live-net-status

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Please use things from https://github.com/openshift/openshift-azure/blob/master/pkg/vmimage/data/usr/local/bin/get-node-logs#L19-L32 and clear that script

Reply:
these are 2 distinct scripts for 2 different purposes, I think they are fine without sharing code between them

Reply:
get-node-logs are not being used and should be deleted and refactored.Networking information in `get-node-logs` is more granual